lndclient/lnd_services.go

```diff
@@ -173,6 +198,9 @@ func NewLndServices(cfg *LndServicesConfig) (*GrpcLndServices, error) {
 			Router:        routerClient,
 			Versioner:     versionerClient,
 			ChainParams:   chainParams,
+			NodeAlias:     nodeAlias,
```

Comment:
This looks like an independent change that could be isolated in a commit?

Reply:
No, we load and store the info we get from `GetInfo` and `GetVersion` in this commit. Or what do you mean?

Reply:
I mean that the `GetInfo` call was already there, so the node alias and key were also already available. Before introducing the version call.

Reply:
Okay, fair enough. I added an extra commit.

Reply:
Maybe it felt a bit over the top, but it is definitely nice now that I am re-reviewing